### ISSUE TITLE
`index.mjs` seems broken

### ISSUE THREAD

**Issue by romainmenke** [postcss/postcss-parser-tests#22]
https://github.com/postcss/postcss-parser-tests/blob/c9f3e77d3d8413378aa5c9a5044c64d82f71dd2c/index.mjs#L6

```
node_modules/postcss-parser-tests/index.mjs:6
export { jsonify, path, each, testOnReal }
                        ^^^^

SyntaxError: Export 'each' is not defined in module
```

full file contents

```js
import testOnReal from './test-on-real.js'
import eachTest from './each-test.js'
import testPath from './test-path.js'
import jsonify from './jsonify.js'

export { jsonify, path, each, testOnReal }
```

**Comment by ai** [comment]
Fixed https://github.com/postcss/postcss-parser-tests/commit/c678f424311d2e175540a64d2ff9443ca38a9c43

The fix was released in 8.5.2.

**Comment by romainmenke** [comment]
Works perfectly! Thank you 🙇
